tests/integration/test_packages.py

```diff
@@ -2,60 +2,62 @@
 
 from os import path
 import pytest
-import tarfile
 
 import utils
 
 
 @pytest.mark.parametrize("env_name", ["without_deps", "with_deps"])
-def test_all_pip_packages(env_name, test_env, tmpdir):
+@pytest.mark.parametrize("env_package", ["pip_packages", "gomod_packages"])
```

Comment:
I will add tests which we expect to fail into "pip_packages" and a few more different tests for "gomod_packages". Can we keep both (env_package and env_name) ?

Reply:
Not sure how these variables are related to tests that are/will be in different files. Could you please explain this comment more?